yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/object_type_mapping.psql

```diff
@@ -17,7 +18,7 @@ JOIN
 LEFT JOIN
     pg_catalog.pg_index pi ON pi.indexrelid = c.oid AND (c.relkind = 'i' OR c.relkind = 'I')
 WHERE 
-    (c.relkind IN ('r', 'p', 'i', 'I', 'm')) AND NOT (c.relkind IN ('i', 'I') AND pi.indisprimary)
+    (c.relkind IN ('r', 'p', 'i', 'I', 'm')) AND NOT (c.relkind ='I' AND pi.indisprimary) -- ignoring partitioned primary keys(since we deal with leave nodes only - having actual data and iops)
```

Comment:
pls create a ticket for handlng this properly for partitioned tables as we had discussed. 
`I` shows up for root partitioned tables' indexes.

Reply:
https://yugabyte.atlassian.net/browse/DB-15159